recipes/pyjulia/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.4" %}
+{% set name = "julia" %}
+
+package:
+  name: pyjulia
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ea83e23a35a4a60014654cbcab70daf1da7539dc2129a19bb7e6354a168bad70
+
+build:
+  skip: True  # [osx or win]
+  number: 0
+  script: python -m pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+    - julia
+  run:
+    - python
+    - julia
+
+test:
+  commands:
+    - python -c 'import julia; julia.Julia()'
+
+about:
+  home: https://github.com/JuliaPy/pyjulia
+  license: MIT
+  license_file: LICENSE
```

Comment:
Please change to `license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'`.

Also please add a comment above this line referencing PR ( https://github.com/JuliaPy/pyjulia/pull/137 ). We can switch to getting it from source once that is merged and released.

Reply:
Should add that doing this should fix the current CI failure.